### PR TITLE
chore: proposal projects deprecation

### PR DESCRIPTION
**Pull request by 1emu**

### CODE REVIEW

File: src/utils/projects.ts [hunk 1]
```diff
@@ -46,7 +47,43 @@ function getFunding(proposal: ProposalAttributes, transparencyVesting?: Transpar
...
+    return ProjectStatus.Finished
+  }
+
+  if (!vesting) {
+    return ProjectStatus.Pending
+  }
+
+  const { status } = vesting
+
+  return toGovernanceProjectStatus(status)
```
Comment (andyesp) [inline src/utils/projects.ts:83]: I'd just write vesting.status here 😅

File: src/utils/projects.ts [hunk 2]
```diff
@@ -46,7 +47,43 @@ function getFunding(proposal: ProposalAttributes, transparencyVesting?: Transpar
...
+
+  return {
+    enacted_at: vesting.start_at,
+    vesting,
+  }
+}
+
+export function getProjectStatus(project: ProjectQueryResult, vesting: VestingWithLogs | undefined) {
+  const legacyCondition = !vesting && project.enacted_description
+  if (project.enacting_tx || legacyCondition) {
```
Comment (andyesp) [inline src/utils/projects.ts:73]: Shouldn't we add the `project.enacting_tx` check to the legacy condition too?

**Review by andyesp — APPROVED** [review]

File: src/utils/projects.ts (hunk 1 above)
Comment (1emu) [reply src/utils/projects.ts:83]: We still need to parse it tho

File: src/services/ProjectService.ts [hunk 3]
```diff
@@ -1,109 +1,144 @@
...
 
 import { BudgetService } from './BudgetService'
 import { ErrorService } from './ErrorService'
 import { ProposalInCreation } from './ProposalService'
 import { VestingService } from './VestingService'
 
-function newestVestingFirst(a: TransparencyVesting, b: TransparencyVesting): number {
-  const startDateSort = new Date(b.vesting_start_at).getTime() - new Date(a.vesting_start_at).getTime()
-  const finishDateSort = new Date(b.vesting_finish_at).getTime() - new Date(a.vesting_finish_at).getTime()
+function newestVestingFirst(a: ProjectInList, b: ProjectInList): number {
```
Comment (andyesp) [inline src/services/ProjectService.ts:43]: `sortByNewestVestingFirst`?

**Review by andyesp — APPROVED** [review]

**Pull request merged**